content/blog/part-5-agentic-ai-team-coordination-mode-in-action.md

```diff
@@ -0,0 +1,180 @@
+from agno.tools.search import DuckDuckGoTools
+from agno.tools.read import Newspaper4kTools
+```
+
+### Searcher Agent
+
+```python
+searcher = Agent(
+    name="Searcher",
+    role="Searches the top URLs for a topic",
+    instructions=[
+        "Generate 3 search terms for a topic.",
+        "Search the web and return 10 high-quality, relevant URLs.",
+        "Prioritize credible sources, suitable for the New York Times."
+    ],
+    tools=[DuckDuckGoTools()],
+    add_datetime_to_instructions=True,
+)
+```
+
+### Writer Agent
+
+```python
+writer = Agent(
+    name="Writer",
+    role="Writes a high-quality article",
+    description="Senior NYT writer tasked with long-form editorial content.",
+    instructions=[
+        "Read all articles using `read_article`.",
+        "Write a structured, engaging article of at least 15 paragraphs.",
+        "Support arguments with factual citations and ensure clarity.",
+        "Never fabricate facts or plagiarize content."
+    ],
+    tools=[Newspaper4kTools()],
+    add_datetime_to_instructions=True,
+)
+```
+
+### Editor Team (Manager Agent in Coordinate Mode)
+
+```python
+editor = Team(
+    name="Editor",
+    mode="coordinate",
+    model=OpenAIChat("gpt-4o"),
+    members=[searcher, writer],
+    description="You are a senior NYT editor coordinating the team.",
+    instructions=[
+        "Delegate research to the search agent.",
+        "Delegate drafting to the writer.",
+        "Review, proofread, and enhance the final article.",
+        "Maintain NYT-level quality, structure, and tone."
+    ],
+    add_datetime_to_instructions=True,
+    send_team_context_to_members=True,
+    show_members_responses=True,
+    markdown=True,
+)
+```
+
+### Running the Team
+
+```python
+Method 1: Print output directly
+editor.print_response("Write an article about latest developments in AI.")
+
+Method 2: Get raw result
+response = editor.run("Write an article about latest developments in AI.")
+```
+
+### Key Parameters Explained
+
+<table>
+  <thead style="background-color:#f2f2f2">
+    <tr>
+      <th>Parameter</th>
+      <th>Purpose</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>mode="coordinate"</code></td>
+      <td>Enables structured delegation and task flow</td>
+    </tr>
+    <tr>
+      <td><code>members=\\\\[...]</code></td>
+      <td>Assigns role-specific agents</td>
+    </tr>
+    <tr>
+      <td><code>send_team_context_to_members</code></td>
+      <td>Shares global task context with all agents</td>
+    </tr>
+    <tr>
+      <td><code>show_members_responses=True</code></td>
+      <td>Displays each member's intermediate output</td>
+    </tr>
+    <tr>
+      <td><code>add_datetime_to_instructions</code></td>
+      <td>Contextualizes outputs with current date/time</td>
+    </tr>
+  </tbody>
+</table>
+
+## Pro Tip: Define success criteria
+
+Adding success_criteria helps agents align their efforts with measurable outcomes.
```

Comment:
Should there really be an underscore between "success" and "criteria"? I do see it in part of the code below, so it may very well be. Just checking.